docs/source/configure.rst

```diff
@@ -1,14 +1,7 @@
 Configuration
 =============
 
-
-The package contains a configuration file ``config.json``. After the package is installed you can check the initial
-configuration parameters using the command line::
-
-$ sentinelhub.config --show
```

Comment:
This should probably stay in, right?

Reply:
the `--show` functionality?

Reply:
Yes, unless it was removed, now I don't remember anymore.